vms/platformvm/blocks/executor/helpers_test.go

```diff
@@ -207,9 +196,20 @@ func newEnvironment(t *testing.T, ctrl *gomock.Controller) *environment {
 		panic(fmt.Errorf("failed to create mempool: %w", err))
 	}
 
+	res.backend = &executor.Backend{
+		Config:       res.config,
+		Ctx:          res.ctx,
+		Clk:          res.clk,
+		Bootstrapped: res.isBootstrapped,
+		Fx:           res.fx,
+		FlowChecker:  res.utxosHandler,
+		Uptimes:      res.uptimes,
+		Rewards:      rewardsCalc,
+		Mempool:      res.mempool,
```

Comment:
if Mempool is removed from the backend (I think it's cleaner to pass it to block Builder from the constructor), this change can be reverted.

Reply:
I answered it above:https://github.com/ava-labs/avalanchego/pull/1363#discussion_r1180259151

We would end up taking the same mempool + txExecutor args in both blockbuilder/blockexecutor.